docker/Dockerfile

```diff
@@ -36,16 +36,12 @@ RUN apt-get update && \
 	libtool \
 	software-properties-common \
 	redis-server \
-	tcl8.5 
+	tcl8.5
 
 # Clone the Dynomite Git
 RUN git clone https://github.com/Netflix/dynomite.git
```

Comment:
Let's:
1) Point this to our fork
2) Pin it to a version/tag.

It could just actually be something like
```
COPY ../ dynomite dynomite
```

Instead of the git clone since we're sure this Dockerfile will run within the dynomite repo.

Reply:
I realize what they're doing after I posted the PR (and I should check conductor image for the same approach).
The key question here is: do we really need a fork?
At the moment I think it is better to create our own docker image in infra repo. It will be based on this, but it will pull a concrete version from the original repo.

I'll apply the proposed change and also will make the other one - it should be a quick one.